common/src/main/java/org/apache/falcon/persistence/BeanUtils.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.falcon.persistence;
+
+import org.apache.falcon.FalconException;
+import org.apache.falcon.entity.v0.EntityType;
+
+/**
+ * Util methods for Bean classes.
+ */
+public final class BeanUtils {
+
+    private BeanUtils(){}
+
+    /*
+    * Checks wheather Entity Type is in feed or process.
+    *
+    * @param  entityType
+    *         The {@code String} to check it against valid {@code enum} EntityType.
+    * @throws FalconException
+    *         if the entityType is not valid.
+    * */
+
+    public static void checkEntityType(String entityType)throws FalconException {
```

Comment:
After discussed moving it to EntityType as EntityType.getEnum will not serve the complete purpose of the function.